sources/packages/api/src/services/education-program/education-program.service.ts

```diff
@@ -351,7 +351,7 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
     } else {
       // default sort and order
       paginatedProgramQuery.orderBy(
-        `CASE programs.programStatus
+        `CASE programs.program_status
```

Comment:
Please add a TODO for further investigation on why `programs.programStatus` was not translated to the DB column name. This will be taken care is the next PR, right?